src/wrapper/rest.cpp

```diff
@@ -291,6 +291,10 @@ void exposeRest()
            (Ogg::XiphComment *(FLAC::File::*)(bool))
            &FLAC::File::xiphComment,
            xiphComment_overloads()[return_internal_reference<>()])
+      .DEF_SIMPLE_METHOD(pictureList)
+      .DEF_SIMPLE_METHOD(removePictures)
+      .DEF_SIMPLE_METHOD(removePicture)
+      .DEF_SIMPLE_METHOD(addPicture)
```

Comment:
Can we have some Python code testing each of these please? Doesn't need to be complicated, provided it calls it without errors.